config.toml

```diff
@@ -138,6 +138,15 @@ sidebar_menu_compact = true
   [[module.imports]]
     path = "github.com/matrix-org/docsy"
     disable = false
+  # Mounting /assets and /static as assets directories helps Hugo to generate
+  # the correct URL.
+  [[module.mounts]]
+    source = 'assets'
+    target = 'assets'
+  [[module.mounts]]
+    source = 'static'
+    target = 'assets'
```

Comment:
Since we're also mounting both `static` and `assets` to the same place, we should leave a note to say that any files named the same in both directories will conflict (I'm don't believe Hugo will error in an obvious way in that case). In that case, files in `static` will win, since it's mounted second.

Reply:
Oop, indeed we are! Would carrying out that alternative solution mean we'd stop unintentionally serving those files? That sounds preferable?

Reply:
I think so. Everything in `/static` is copied as-is in the rendered website. Files in `/asset` are only in the rendered website if they are in use.

Reply:
I added a commit that does just that, and merged main to solve a conflict because of it.